src/sentry.ts

```diff
@@ -49,7 +49,6 @@ const sentryClient = Sentry.init({
 
 console.log('sentryClient is defined:', !!sentryClient);
 const provider = new NodeTracerProvider({
-    sampler: new SentrySampler(sentryClient!),
```

Comment:
we _can_ skip the sampler if we really don't care about distributed tracing etc. I'd still recommend to wrap your own sampler with `wrapSamplingDecision` but it should generally work this way too.